setup.py

```diff
@@ -25,6 +24,19 @@ def fetch_requirements(path):
         return [r.strip() for r in fd.readlines()]
 
 
+def available_vector_instructions():
+    cpu_vector_instructions = {}
+    try:
+        import cpufeature
+        cpu_vector_instructions = cpufeature.CPUFeature
+    except _:
+        warnings.warn(
```

Comment:
Any idea if this will catch a segfault in the import? Otherwise we may need a signal handler instead :-(.

Reply:
Unfortunately, I don't know the answer to this question, especially since we can't repro the segfault. I guess I could create a nullptr deference in a C++ extension to test it.

Reply:
I can't figure out how a signal handler in python can proceed after catching a segfault. Right now it simply replays the faulting instruction in the extension. I will punt on this for now.